<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="VideoWidget" name="widget_videoFront" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px dashed black;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_videoSide" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border: 3px dashed black;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3Dplot" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border: 3px dashed black;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_front">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_side">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseFile_videoFront">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Chose File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseFile_videoSide">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Chose File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Side</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed black;
			
	font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Save Video</string>
    </property>
    <widget class="QPushButton" name="pushButton_SaveVideo_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SaveVideo_pause">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SaveVideo_end">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed black;
			
	font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Reason</string>
    </property>
    <widget class="QPushButton" name="pushButton_reason_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reason_pause">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reason_end">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chooseFile_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 2px solid black;
			
	font: 75 14pt &quot;Arial&quot;;
	border-radius: 8px; /* 设置圆角 */

}</string>
     </property>
     <property name="text">
      <string>Load Model</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 2px solid black;
			
	font: 75 14pt &quot;Arial&quot;;
	border-radius: 8px; /* 设置圆角 */

}</string>
    </property>
    <property name="text">
     <string>Synthesis</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>380</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed black;
			
	font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Data Save</string>
    </property>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { border: 1px solid black;
			
	font: 75 12pt &quot;Arial&quot;;
}</string>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_startPlay">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 2px solid black;
			
	font: 75 14pt &quot;Arial&quot;;
	border-radius: 8px; /* 设置圆角 */

}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pausePlay">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 2px solid black;
			
	font: 75 14pt &quot;Arial&quot;;
	border-radius: 8px; /* 设置圆角 */

}</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actiontheme"/>
   </widget>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontheme">
   <property name="text">
    <string>theme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
